src/components/sendAskMoney/SendAskMoney.vue

```diff
@@ -566,10 +566,22 @@
             }
             this.selectedCreditAccount = this.creditableMoneyAccounts[0]
           }
-          let url = await this.selectedCreditAccount._obj.getCreditUrl(
-            this.amountForCredit
-          )
-          this.urlForHyperlink = url.order_url
+          try {
+            let url = await this.selectedCreditAccount._obj.getCreditUrl(
+              this.amountForCredit
+            )
+            this.urlForHyperlink = url.order_url
+          } catch (error) {
+            console.log(error)
+            this.$Swal.fire({
+              position: "top",
+              icon: "error",
+              title:
+                "Il y a eu un problème lors de la tentative de creditation de votre compte",
```

Comment:
"créditation" > "crédit"
Ou bien "Il y a eu un problème en tentant de créditer votre compte"

Reply:
Done, thank you for the review